src/main/java/org/folio/circulation/item/service/CirculationItemService.java

```diff
@@ -0,0 +1,85 @@
+package org.folio.circulation.item.service;
+
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.folio.circulation.item.domain.entity.Item;
+import org.folio.circulation.item.domain.mapper.CirculationItemMapper;
+import org.folio.circulation.item.exception.IdMismatchException;
+import org.folio.circulation.item.exception.ResourceAlreadyExistException;
+import org.folio.circulation.item.repository.CirculationItemRepository;
+import org.folio.rs.domain.dto.CirculationItem;
+import org.folio.spring.exception.NotFoundException;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class CirculationItemService {
```

Comment:
Please create a interface and implements it in this class

Reply:
done